scripts/ci/selective_tests.sh

```diff
@@ -58,28 +60,83 @@ function output_all_basic_variables() {
     initialization::ga_output kubernetes-exclude '[]'
 }
 
-function set_outputs_run_all_tests() {
+function run_tests() {
     initialization::ga_output run-tests "true"
-    initialization::ga_output run-kubernetes-tests "true"
-    initialization::ga_output test-types "Core Other API CLI Providers WWW Integration Heisentests"
 }
 
-function set_output_skip_all_tests() {
+function skip_running_tests() {
     initialization::ga_output run-tests "false"
+}
+
+function run_kubernetes_tests() {
+    initialization::ga_output run-kubernetes-tests "true"
+}
+
+function skip_running_kubernetes_tests() {
     initialization::ga_output run-kubernetes-tests "false"
+}
```

Comment:
How about.

```suggestion
function run_kubernetes_tests() {
    initialization::ga_output run-kubernetes-tests "${1:-true}"
}
```

And then we can just call it as:

```diff
-if [[ ${kubernetes_tests_needed} == "true" ]]; then
-    run_kubernetes_tests
-else
-    skip_running_kubernetes_tests
-fi
+run_kubernetes_tests "${kubernetes_tests_needed}"
```

Reply:
(Or maybe `set_run_kubernetes_tests "${kubernetes_tests_needed}"` might be slightly clearer)

Reply:
Right. Will do it. I was doing it at 1am :)